lib/pure/options.nim

```diff
@@ -23,38 +23,38 @@
 ## Let's start with an example: a procedure that finds the index of a character
 ## in a string.
 ##
-## .. code-block:: nim
-##
-##   import options
-##
-##   proc find(haystack: string, needle: char): Option[int] =
-##     for i, c in haystack:
-##       if c == needle:
-##         return some(i)
-##     return none(int)  # This line is actually optional,
-##                       # because the default is empty
-##
-## .. code-block:: nim
-##
-##    let found = "abc".find('c')
-##    assert found.isSome and found.get() == 2
-##
+runnableExamples:
```

Comment:
instead of duplicating this example, just refer to it:

```
## in a string. See example below.
##
## The `get` operation demonstrated above returns the underlying value, or
## ...
## How to deal with an absence of a value:

runnableExamples:
  ...

```

Reply:
how can I MAKE them pass example tests?